src/Redux/Actions/NoteAction.js

```diff
@@ -12,3 +12,10 @@ export const sortNote = () => {
     type: actionTypes.SORT_NOTE,
   };
 };
+
+export const updateNote = (newNote) => {
+  return {
+    type: actionTypes.UPDATE_NOTE,
+    newNote,
```

Comment:
you should keep this consistent with other actions that have data and name them `value` as that is what you have for creating a new note. Keeping consistency on how things are named helps with readability